<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>82</y>
      <width>118</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>82</y>
      <width>118</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="switchPlayer1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchPlayer2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameTime1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>120 sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameTime2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5 min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP GAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameInfo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
